docs/pages/database-access/guides/sql-server-ad-pkinit.mdx

```diff
@@ -0,0 +1,387 @@
+---
+title: Database Access with Microsoft SQL Server with Active Directory PKINIT authentication (Preview)
```

Comment:
With the standard `| Teleport Docs` suffix that gets added to the H1, the `title` tag's text is 112 characters. The rule of thumb for SEO is around 60 characters, though I'm not sure how often we follow this. In any case, we should find a way to shorten the title.

Per our style guide, we also use title case in titles, so we should capitalize "authentication"